package.json

```diff
@@ -103,24 +99,12 @@
     "prettier": "^3.6.2",
     "publint": "^0.2.12",
     "rimraf": "^4.4.1",
-    "run-all": "^1.0.1",
     "simple-git-hooks": "^2.13.1",
     "tsup": "^8.5.0",
-    "tsx": "^3.14.0",
+    "tsx": "^4.20.6",
     "typescript": "^5.9.2",
     "vitest": "^1.6.1"
```

Comment:
We're on vitest 3 now, nearly vitest 4, but that's _definitely_  out of scope for this PR.